cloud/edgecontroller/pkg/controller/constants/service.go

```diff
@@ -0,0 +1,32 @@
+package constants
+
+// Service level constants
+const (
+	// module
+	EdgeControllerModuleName     = "controller"
+	CloudHubControllerModuleName = "cloudhub"
+
+	// Resource sep
+	ResourceSep               = "/"
+	ResourceSliceLength        = 5
+	ResourceSliceLengthQuery   = 4
+	ResourceNodeIndex         = 0
+	ResourceNodeIDIndex       = 1
+	ResourceNamespaceIndex    = 2
+	ResourceResourceTypeIndex = 3
+	ResourceResourceNameIndex = 4
+	ResourceNode              = "node"
+
+	// Group
+	GroupResource = "resource"
+
+	// Nvidia Constants
+	// NvidiaGPUStatusAnnotationKey is the key of the node annotation for GPU status
+	NvidiaGPUStatusAnnotationKey = "huawei.com/gpu-status"
+	// NvidiaGPUDecisionAnnotationKey is the key of the pod annotation for scheduler GPU decision
+	NvidiaGPUDecisionAnnotationKey = "huawei.com/gpu-decision"
```

Comment:
can we avoid the reference ?

Reply:
Currently, our edge supports integration with huawei cloud. We will remove these references maybe later.